truescrub/api.py

```diff
@@ -21,6 +21,17 @@
 
 SHARED_KEY = os.environ.get('TRUESCRUB_KEY', 'afohXaef9ighaeSh')
 
+zmq_socket = zmq.Context().socket(zmq.PUSH)
```

Comment:
Slightly worried that this might be initializing too early (when the script is processed, before it's executed), and should be moved to an init function.

Reply:
It just happens at module import time. There's no connection happening until the `connect` call.